src/input-login.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Text, TextInput, StyleSheet } from "react-native";
+
+interface InputProps {
+	inputHeader: string;
+	handleTextChange: Function;
+	value: string;
+	secureTextEntry?: boolean;
+}
+
+function InputLogin (props: InputProps): JSX.Element {
+  return (
+    <React.Fragment>
+			<Text>
+				{props.inputHeader}
+			</Text>
+    	<TextInput
+    	  style={styles.input}
+    	  value={props.value}
+				onChangeText={(value) => props.handleTextChange(value)}
```

Comment:
nesses casos onde a função passada é o mesmo tipo da esperada a gente não precisa declarar uma função nova, basta passar a que já existe mesmo
```suggestion
				onChangeText={props.handleTextChange}
```